{"uuid": "54e82cf5-8f9c-4d7b-8cf5-9bdc57409073", "children": ["a3ad5c8a-5d08-401f-90fe-efa5ddab5a4e", "ee3a20ad-dfd4-4827-a205-8f95d4816d65", "71f4204b-395f-4d86-982e-5af5559f149c", "44c0c0b1-b095-4694-bb89-dc79243f588b"], "befores": [{"name": "_Module__pytest_setup_module", "status": "passed", "start": 1619518824831, "stop": 1619518824831}], "afters": [{"name": "_Module__pytest_setup_module::0", "status": "broken", "statusDetails": {"message": "pymysql.err.InternalError: (1105, \"Unknown column 'customer_third_id' in 'where clause'\")\n", "trace": "  File \"C:\\Users\\chenping\\PycharmProjects\\apitest\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\Users\\chenping\\PycharmProjects\\apitest\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 800, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\chenping\\PycharmProjects\\apitest\\venv\\lib\\site-packages\\_pytest\\python.py\", line 474, in xunit_setup_module_fixture\n    _call_with_optional_argument(teardown_module, request.module)\n  File \"C:\\Users\\chenping\\PycharmProjects\\apitest\\venv\\lib\\site-packages\\_pytest\\python.py\", line 634, in _call_with_optional_argument\n    func()\n  File \"C:\\Users\\chenping\\PycharmProjects\\apitest\\testcase\\Loginapi\\Find_Bind_Third_User_test.py\", line 66, in teardown_module\n    custid=PyMySQL().mysqlget(sql)\n  File \"C:\\Users\\chenping\\PycharmProjects\\apitest\\utils\\pysql.py\", line 90, in mysqlget\n    cursor.execute(sql)\n  File \"C:\\Users\\chenping\\PycharmProjects\\apitest\\venv\\lib\\site-packages\\pymysql\\cursors.py\", line 170, in execute\n    result = self._query(query)\n  File \"C:\\Users\\chenping\\PycharmProjects\\apitest\\venv\\lib\\site-packages\\pymysql\\cursors.py\", line 328, in _query\n    conn.query(q)\n  File \"C:\\Users\\chenping\\PycharmProjects\\apitest\\venv\\lib\\site-packages\\pymysql\\connections.py\", line 517, in query\n    self._affected_rows = self._read_query_result(unbuffered=unbuffered)\n  File \"C:\\Users\\chenping\\PycharmProjects\\apitest\\venv\\lib\\site-packages\\pymysql\\connections.py\", line 732, in _read_query_result\n    result.read()\n  File \"C:\\Users\\chenping\\PycharmProjects\\apitest\\venv\\lib\\site-packages\\pymysql\\connections.py\", line 1075, in read\n    first_packet = self.connection._read_packet()\n  File \"C:\\Users\\chenping\\PycharmProjects\\apitest\\venv\\lib\\site-packages\\pymysql\\connections.py\", line 684, in _read_packet\n    packet.check_error()\n  File \"C:\\Users\\chenping\\PycharmProjects\\apitest\\venv\\lib\\site-packages\\pymysql\\protocol.py\", line 220, in check_error\n    err.raise_mysql_exception(self._data)\n  File \"C:\\Users\\chenping\\PycharmProjects\\apitest\\venv\\lib\\site-packages\\pymysql\\err.py\", line 109, in raise_mysql_exception\n    raise errorclass(errno, errval)\n"}, "start": 1619518825190, "stop": 1619518825301}], "start": 1619518824831, "stop": 1619518825301}